apps/ewallet/lib/mix/tasks/omg.seed.ex

```diff
@@ -95,6 +99,9 @@ defmodule Mix.Tasks.Omg.Seed do
 
   defp assume_yes?([]), do: false
 
+  # disable prompt when running blockchain seeds
+  defp assume_yes?(["--blockchain" | _t]), do: true
```

Comment:
Do we need this? I won't to avoid workaround if we can. Since the prompt depend on each seed and blockchain doesn't have one, it shouldn't have a prompt already... I think.

Reply:
woops, we don't need it anymore 🤦‍♂

Reply:
Hmm, actually I tested again and if removed, we are prompted to press enter ...